packages/lexical-playground/src/plugins/ToolbarPlugin.css

```
@@ -12,3 +12,14 @@
   justify-content: right;
   margin-top: 20px;
 }
+
+.ToolbarPlugin__dialogImageActions {
```

Just making it a little more general as I think we'll use this styling again:

```suggestion
.ToolbarPlugin__dialogButtonsList {
```